scheduler/scheduler.go

```diff
@@ -246,6 +236,86 @@ func NewMesosSchedulerDriver(config DriverConfig) (initializedDriver *MesosSched
 	return
 }
 
+func (driver *MesosSchedulerDriver) makeWithScheduler(cs Scheduler) func(func(Scheduler)) {
```

Comment:
Yeah, I totally understand. Let's not put too much brain cycles here. They'll be better spent on the new bindings.